### ISSUE TITLE
Error with webpack

### ISSUE THREAD

**Issue by thibaudsalsa** [enquirer/enquirer#431]
Hello everyone, 

I don't know that much your project but I saw webpack is using it.
on the 07/26/2023 we deploy our backend on AWS and today 07/27/2023 we seem to have this error.

```node_modules/enquirer/lib/types/string.js:15```
```this.keypressTimeout = this.options.keypressTimeout ?? null;```
```SyntaxError: Unexpected token '?'```
We don't have any changes since the last deployment but I saw that this line was update 15h ago.

this is are setup for the dockerfile
```FROM --platform=linux/amd64 node:13.14-buster```
and tsc -v
Version 4.2.4
Thanks a lot for any answer !

**Comment by chenjiahan** [comment]
**enquirer 2.4.0 cannot run in Node.js <= 16 because it uses incompatible syntax.**

@jonschlinkert Can you revert these changes and release them in 3.0.0?

```bash
(node:80580) UnhandledPromiseRejectionWarning: /pnpm/enquirer@2.4.0/node_modules/enquirer/lib/types/string.js:33
      this.state.multilineBuffer ||= '';
                                 ^^^

SyntaxError: Unexpected token '||='
    at wrapSafe (internal/modules/cjs/loader.js:1001:16)
    at Module._compile (internal/modules/cjs/loader.js:1049:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
```

**Comment by targeral** [comment]
I'm having the same problem.

**Comment by vaimdevs** [comment]
v2.3.6 work fine for me

**Comment by jonschlinkert** [comment]
Closing this was fixed by [2.4.1](https://github.com/enquirer/enquirer/commit/70bdb0fedc3ed355d9d8fe4f00ac9b3874f94f61). thanks for creating the issue.
